boards/firechip/base-workloads/fedora-base/overlay/etc/yum.repos.d/fedora-riscv.repo

```diff
@@ -1,21 +1,17 @@
 [fedora-riscv]
 name=Fedora RISC-V
-baseurl=http://fedora-riscv.tranquillity.se/repos-dist/f29/latest/riscv64/ https://dl.fedoraproject.org/pub/alt/risc-v/repo/fedora/29/latest/riscv64/ https://mirror.math.princeton.edu/pub/alt/risc-v/repo/fedora/29/latest/riscv64/
+baseurl=http://fedora.riscv.rocks/repos-dist/rawhide/latest/riscv64/
```

Comment:
The backups were originally added because the main repo kept going down. I guess the official fedora repo is probably reliable enough. Are there other mirrors we could add just in case?

Reply:
I'm not aware of any.  fedora.riscv.rocks is used by Red Hat for nightly builds.  The "main" repo (https://dl.fedoraproject.org/pub/alt/risc-v/repo/fedora/rawhide/latest/riscv64/) hasn't been updated since mid-2019.